frontend/screens/CreateAccount.js

```diff
@@ -277,21 +255,25 @@ createAccount.propTypes = {
   navigation: PropTypes.object,
 }
 const styles = StyleSheet.create({
+  title:
+  {
+    fontSize: normalize(25),
+    marginTop: '10%',
+    marginBottom: '5%',
+  },
   button: {
     borderColor: hex,
```

Comment:
try to use the hex from global styles...style sheet is an object so u can just do like hex.color to get the string